tools/scenario-player/scenario_player/utils.py

```diff
@@ -71,8 +71,7 @@ def getlines(self, start, stop=None):
 
 
 class ConcatenableNone:
-    def __radd__(self, other):
-        return other
+    msg = ''
```

Comment:
What is this good for?

Reply:
the scenario player always flickers in any of may and Karl's terminals when we do not change that.

Reply:
That class is an ugly hack to enable the log handling in the scenario player. 
The stdlib logging module's base handler used to transparently call `write()` on whatever the formatter output as the `record`. 

The SP 'abuses' that by passing the raw `LogRecord` instances to a fake stream object in order to render the log into the urwid pane. (see `scenario_player.ui::NonStringifyingProcessorFormatter / _DummyFormatter`)

In Python 3.7 an optimization was added to logging that concatenates the terminator (usually `\n`) to the record before calling write. This class simply returns the other argument.

Dominik and Karl (I believe) had issues with exceptions complaining that this class has no `msg` attribute. I was meaning to look into why that's happening.

Reply:
I uncluded that back in an will simply change this locally